### DISCUSSION TITLE
how can i enable infinite scrolling?

### DISCUSSION THREAD

**Discussion by useronegit** [searxng/searxng#1333] [Q&A]
how can i enable infinite scrolling by default?
i can't find this option in settings.yml

i use searxng-docker

**Comment by mrpaulblack** [comment]
Hi @useronegit ,
You can enable inifintie scrolling as a setting in https://github.com/searxng/searxng-docker/blob/master/searxng/settings.yml .

The setting is really hidden here: https://github.com/searxng/searxng/blob/8177bf3f0a4d4f22cf63812dc86a80535cd15d68/searx/settings_defaults.py#L189

It should be possible for you to add this to your settings.yml (not tested) here -> https://github.com/searxng/searxng-docker/blob/b1b174a3392275f0c9d323c82ec9d45055ac124a/searxng/settings.yml#L8:

```
infinite_scroll: true
```

Did this help?
